.github/workflows/download_pipeline.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Install Nextflow
-        uses: nf-core/setup-nextflow@b9f764e8ba5c76b712ace14ecbfcef0e40ae2dd8 # v1
+        uses: nf-core/setup-nextflow@v1
```

Comment:
```suggestion
        uses: nf-core/setup-nextflow@v2
```
not sure if the linting then complains about it... only one way to find out 🙂

Reply:
sure.

Reply:
this one didn't work, but the other one.

Reply:
whoopsy, let me try again